cmake/configure.cmake

```diff
@@ -51,6 +51,16 @@ if(WIN32)
   endif(NOT MSVC)
 endif(WIN32)
 
+if(WITH_MUSL)
+    add_definitions(-DPADDLE_WITH_MUSL)
+
+    message(STATUS, "WITH_MKL=OFF, when WITH_MUSL=ON")
```

Comment:
recommended to write more like an English message, such as `Set conpile option WITH_MKL=OFF when compile option WITH_MUSL=ON`

Reply:
ready in 2e12aebf246b69294ff57ee797105b43ab288b0e